coal_petroleum/data/ir_model_fields.xml

```diff
@@ -0,0 +1,228 @@
+    </record>
+
+    <record id="new_decimal_gas_qual_fb613a78-4a66-4719-a4e9-fb135ff8c65f" model="ir.model.fields">
+        <field name="name">x_aki</field>
+        <field name="model">x_quality_check_worksheet_template_2_studio</field>
+        <field name="model_id" ref="model_worksheet_template_2"/>
+        <field name="ttype">float</field>
+        <field name="field_description">AKI</field>
+    </record>
+
+    <record id="new_decimal_purchase_222506a8-b8a5-48ab-8ac2-a6e49cbb4596" model="ir.model.fields">
+        <field name="name">x_tolerance</field>
+        <field name="model">purchase.order.line</field>
+        <field name="model_id" ref="purchase.model_purchase_order_line"/>
+        <field name="ttype">float</field>
+        <field name="field_description">Tolerance</field>
+    </record>
+
+    <record id="new_decimal_stock_mo_1490dcfd-3d4f-4409-9ade-f8ab83dae3f6" model="ir.model.fields">
+        <field name="name">x_po_tolerance</field>
+        <field name="model">stock.move</field>
+        <field name="model_id" ref="stock.model_stock_move"/>
+        <field name="ttype">float</field>
+        <field name="field_description">PO Tolerance</field>
+    </record>
+
+    <record id="new_decimal_stock_mo_d2578e16-de35-4d6d-88ea-2b36eddf2c8a" model="ir.model.fields">
+        <field name="name">x_allowed_quantity</field>
+        <field name="model">stock.move</field>
+        <field name="model_id" ref="stock.model_stock_move"/>
+        <field name="ttype">float</field>
+        <field name="field_description">Allowed Quantity</field>
+    </record>
+
+    <record id="new_related_field_st_03e80340-8939-4da1-813a-927b5e121571" model="ir.model.fields">
+        <field name="name">x_po_tolerance_1</field>
+        <field name="model">stock.move</field>
+        <field name="model_id" ref="stock.model_stock_move"/>
+        <field name="ttype">float</field>
+        <field name="field_description">Po Tolerance</field>
+        <field name="related">purchase_line_id.x_tolerance</field>
+    </record>
+
+    <record id="new_selection_coal_q_9d096caf-1b8a-43ff-a8cb-d5977dacdca1" model="ir.model.fields">
+        <field name="name">x_test_type_1</field>
+        <field name="model">x_quality_check_worksheet_template_1_studio</field>
+        <field name="model_id" ref="model_worksheet_template_1"/>
+        <field name="ttype">selection</field>
+        <field name="field_description">Test Type</field>
+        <field name="selection">[('adb', 'ADB - Air-Dried Basis'), ('arb', 'ARB - As-Received Basis'), ('db', 'DB-Dry-Basis')]</field>
+    </record>
+
+    <record id="new_text_transfer_33a74c56-96a3-4912-8a83-17d22d7a0d57" model="ir.model.fields">
+        <field name="name">x_vehicle_no</field>
+        <field name="model">stock.picking</field>
+        <field name="model_id" ref="stock.model_stock_picking"/>
+        <field name="ttype">char</field>
+        <field name="field_description">Vehicle No</field>
```

Comment:
No, Vehicle is not mandatory for every transfer

Reply:
But then the knowledge needs to be updated as it mentions 

> It is required to provide the vehicle number for the purpose of formal documentation

Reply:
As per task owner, 
If I make this field mandatory then it will be visible in all transfers. In case of internal transfer there will be no vehicle. 

We need to put attrs in view or domain in the field and make this field mandatory.

Reply:
@vava-odoo I have updated the code based on the conversation.

Thank you!